doc/bridge_packages/flame_forge2d/joints.md

```diff
@@ -112,3 +112,45 @@ to 0, the higher the value, the less springy the joint becomes.
 ```{warning}
 Do not use a zero or short length.
 ```
+
+
+### `FrictionJoint`
+
+A `FrictionJoint` is used for top-down friction. It provides 2D translational friction and angular
+friction.
+
+The `initialize` method requires two bodies and an anchor. The first body is the body the friction
+will be applied to. The second body can be any body in the world, like a floor or an invisible
+body. The reason is that the friction force will be applied to the first body when it collides with
+any other bodies, not only the second body of the joint.
```

Comment:
why do we need to specify a second body then? can we make it more clear, this seems to imply that the second body is irrelevant

Reply:
Good catch. I completely misunderstood that this joint is designed for top-down games, and wrote this piece based on that assumption. Fixed now